pkgs/applications/networking/browsers/palemoon/default.nix

```diff
@@ -202,7 +201,6 @@ stdenv.mkDerivation rec {
     done
 
     # Remove unneeded SDK data from installation
-    # TODO: move to a separate output?
     rm -rf $out/{include,share/idl,lib/palemoon-devel-${version}}
```

Comment:
should this go into a "dev" output?

Reply:
https://forum.palemoon.org/viewtopic.php?f=37&t=26796&p=214676#p214729

Upstream devs said it's of not much use to anyone and I should just remove it. Their prebuilt packages don't include it either.

Reply:
@OPNA2608 what do you think to add this info as a small comment?